apps/api/src/person/dto/update-person.dto.ts

```diff
@@ -1,14 +1,56 @@
+import { ApiProperty } from "@nestjs/swagger"
+import { Expose } from "class-transformer"
+import { IsString } from "class-validator"
+
 export class UpdatePersonDto {
+  @ApiProperty()
+  @Expose()
+  @IsString()
   firstName?: string
+
+  @ApiProperty()
+  @Expose()
+  @IsString()
   lastName?: string
+
+  @ApiProperty()
+  @Expose()
+  @IsString()
```

Comment:
Let's switch the `@IsString()` decorator for more specific `@IsEmail` decorator.

Otherwise users will be able to add any gibberish string in the email field (if DB let's them)